assets/src/edit-story/components/colorPicker/karma/eyedropper.karma.js

```diff
@@ -52,23 +52,28 @@ describe('Eyedropper', () => {
   it('should get color from the image to page background', async () => {
     // Insert image that will be the color source
     const image = fixture.editor.library.media.item(1);
-    const canvas = fixture.editor.canvas.framesLayer.fullbleed;
-    await fixture.events.mouse.seq(({ down, moveRel, up }) => [
-      moveRel(image, 1, 1),
-      down(),
-      moveRel(canvas, 30, 30),
-      up(),
-    ]);
+    // Only regestering image click by clicking twice for some reason
+    await fixture.events.click(image);
+    await fixture.events.click(image);
```

Comment:
Yeah, this is the issue we've also recorded in #7481 and that we need to fix in a general manner.